### PR TITLE
Feature | Added dataUrl for Response

### PR DESCRIPTION
**Pull request by guanguans**

### CODE REVIEW

**Comment by Sammyjo20** [comment]
Hey, @guanguans thank you for submitting this PR - I think this is a cool addition to Saloon! I'm going to have a quick look over the PR but I think the implementation is solid.

File: src/Http/Response.php [hunk 1]
```diff
@@ -333,6 +333,14 @@ public function dom(): Crawler
         return new Crawler($this->body());
     }
 
+    /**
+     * Convert the response to a data URL
+     */
+    public function dataUrl(): string
+    {
+        return 'data:'.$this->psrResponse->getHeaderLine('content-type').';base64,'.base64_encode($this->body());
```
Comment (Sammyjo20) [inline src/Http/Response.php]:
```suggestion
        return 'data:'.$this->psrResponse->getHeaderLine('Content-Type').';base64,'.base64_encode($this->body());
```

**Review by Sammyjo20 — CHANGES_REQUESTED** [review]
Just one suggestion - would it be better to used the camel-case variant for `Content-Type`? I guess it's up to the PSR-7 implementation behind the scenes on how it handles it, but I'm pretty sure Guzzle normalises it anyway.

File: src/Http/Response.php (hunk 1 above)
Comment (guanguans) [reply src/Http/Response.php]: 👍 It has been fixed.

**Review by Sammyjo20 — APPROVED** [review]
Looks good! I will release this later. Thanks again!

**Pull request merged**